scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
+
+
+OP="$1"
+CHAIN="RH-Firewall-1-INPUT"
+POSITION=$(iptables -L --line-n -n | grep -A 11 "Chain $CHAIN" | grep "443" | cut -d " " -f1)
```

Comment:
The use of `grep -A 11` is fragile.
Instead you should specify the relevant chain:
`iptables -L 'RH-Firewall-1-INPUT' --line-n -n` or equivalently `iptables -L "${CHAIN}" --line-n -n`